internal/catalog/internal/types/dns_policy_test.go

```diff
@@ -161,3 +162,20 @@ func TestValidateDNSPolicy_EmptySelector(t *testing.T) {
 	require.ErrorAs(t, err, &actual)
 	require.Equal(t, expected, actual)
 }
+
+func TestDNSPolicyACLs(t *testing.T) {
+	// Wire up a registry to generically invoke hooks
+	registry := resource.NewRegistry()
+	RegisterDNSPolicy(registry)
```

Comment:
Is this registry ever used?

Reply:
DNSPolicy is currently not registered because it's not implemented and so I needed to register it explicitly. It's not used, but I added ACLs for it anyways for whenever we do implement it.